InterviewFlashCards/QuestionsViewController.swift

```diff
@@ -54,23 +47,21 @@ class QuestionsViewController: UIViewController {
     }
 
     override func prepare(for segue: UIStoryboardSegue, sender: Any!) {
-        if (segue.destination is IFCAnswerViewController) {
-            (segue.destination as! IFCAnswerViewController).flashCard = flashCards[currentIndex]
+        if segue.identifier == segueIdentifier {
+            (segue.destination as! AnswersViewController).flashCard = flashCards[currentIndex]
```

Comment:
```if let identifier = segue.identifier {
            (segue.destination as! AnswersViewController).flashCard = flashCards[currentIndex]
        }```

hm, i've never used `if let` with segues, but what am i missing? is force downcasting that bad  if we know for sure that we'll definitely have something to segue to?